### ISSUE TITLE
Export :global classes?

### ISSUE THREAD

**Issue by sprat** [webpack-contrib/css-loader#351]
I've noticed that `:global` classes are not exported when we require the CSS file. Is there a way/an option to allow global exports?

Thanks

**Comment by faceyspacey** [comment]
agreed. i'm not sure why i can't access them on `styles.someClass` just like local ones. Without this, the result is using multiple paradigms to access classes in your React components:

```
<div className={`${styles.someClass} some-global`} />
```

While that's not horrible, what is though is if you're using React Native Web and you expect all your styles on an importable `styles` object because you can't use className strings. If globals were properly assigned to the exported `styles` object you could simply use different styles between a React Native project and a React Native Web Project, while keeping the components the same. 

That said, you also have to do the following thing I do:

`<View style={[styles.someClass, styles.someGlobal]} />`

I.e. you don't use the standard RNW `View` component. You roll your own that accepts class strings (possibly within an array) and if it's a string moves it to the `className` prop or if it's an object treats it as an inline style. 

In short, it doesn't seem like it would hurt any backwards compatibility to just export globals as well. If apps aren't using them, nothing changes for them. It's just unused keys on the `styles` object. For people that were hoping to use them, it's a blessing and basically expected.

**Comment by evilebottnawi** [comment]
Feel free to PR

**Comment by evilebottnawi** [comment]
let's move this in one issue https://github.com/webpack-contrib/css-loader/issues/351
